src/client/ui/panels/HierarchyPanelContainer.js

```diff
@@ -195,10 +195,10 @@ class HierarchyPanelContainer extends Component {
   renderHierarchyNodeMenu = props => {
     const node = props.trigger;
     const hasParent = node && node.object.parent;
+    if (!hasParent) return null;
```

Comment:
One reason we might not want to hide the Scene node is that we still might want to give users the ability to mark the entire scene as "Static". The default is "Dynamic". But perhaps we can do a reasonable default thing here instead.